.codespellrc

```diff
@@ -0,0 +1,5 @@
+[codespell]
+skip = .git,*.pdf,*.svg,go.sum,.codespellrc,vendor
+check-hidden = true
+# ignore-regex = 
```

Comment:
git whitespace checks complaining about an extra space after the `=` on line 4.

You should be able to verify with: `git show --check 045a8a04e6354ac102b4ad8a46857e5c8d988edd`

Reply:
thanks, fixed, rewritten (why to breed commits) and force pushed after re-signing-off